content/en/docs/tutorials/stateless-application/expose-external-ip-address.md

```diff
@@ -50,7 +39,6 @@ external IP address.
 kubectl apply -f https://k8s.io/examples/service/load-balancer-example.yaml
 ```
```

Comment:
👋 @RA489 . I just looked at the preview and noticed that the first item in the list does not indent the "codenew" shortcode under the item starting with "Run a Hello World ...". The code block following this shortcode should also be indented under "Run a Hello World ...". It appears that the numbered list is not rendering completely.
I suggest to check the indentation of line 42 which starts with, "The preceding command creates ...".

Reply:
sure

Reply:
I will update accordingly.

Reply:
@kbhawkey updated the indentation. PTAL.

Reply:
No. Line 36 and the following lines are still not properly indented, so the result is incorrect.

https://deploy-preview-25080--kubernetes-io-master-staging.netlify.app/docs/tutorials/stateless-application/expose-external-ip-address/

Reply:
@RA489 , Could you:
- indent line 36 the same way you indented line 52
- indent lines 38-40 the same way you indented line 52
These code blocks should appear under the `R` in `Run a Hello World application in your cluster:`
OK?